misk-grpc-tests/build.gradle

```diff
@@ -28,9 +29,15 @@ protobuf {
   }
 }
 
+wire {
+  kotlin {
+  }
+}
```

Comment:
Does this make the java code stop getting generated? Or does it just add the kotlin version?

Reply:
This merely adds the Kotlin sources. Tests are changed to ensure we're using those and not the Java ones.

Reply:
And we deleted the Java code!

Reply:
Delete this too?

misk-grpc-tests/build-wire.sh